app/connectors/ApiConnector.scala

```diff
@@ -35,18 +37,24 @@ trait ApiConnector {
   def helloApplication(): Future[JsValue] = api("/hello/application", Some(appToken))
 
   private def api(endpoint: String, token: Option[String] = None): Future[JsValue] = {
-    val headers: Seq[(String, String)] = token match {
-      case Some(t) => Seq("Authorization" -> s"Bearer $t", "Accept" -> versionHeader)
-      case None => Seq("Accept" -> versionHeader)
+    val authorizationHeaders: Seq[(String, String)] = token match {
+      case Some(t) => Seq("Authorization" -> s"Bearer $t")
+      case None => Seq()
     }
-    extractJson[JsValue](
-      WS.url(s"$serviceUrl$endpoint").withHeaders(headers:_*).get()
+    val headers = authorizationHeaders ++ Seq(
+      "Accept" -> versionHeader,
+      "User-Agent" -> ApplicationContext.appName,
```

Comment:
Remove these two headers from here
This is just an example client, so it does not have to abide the HMRC standards